tests/test_linear_congruential_random_generator.c

```diff
@@ -15,10 +15,18 @@ int main(void) {
   int i, n = 100;
 
   printf("linear congruential random generator\n");
-  for (i = 0; i < n; i++)
-    printf("%1.3f ", linear_congruential_random_generator());
-  printf("\n");
-  return (0);
+  for (i = 0; i < n; i++) {
```

Comment:
I don't get why there is a loop here? Is there a specific reason why it runs 100 times?